zilliqa/src/exec.rs

```diff
@@ -158,7 +158,7 @@ impl State {
         &self,
         from_addr: Address,
         to_addr: Option<Address>,
-        gas_price: u64,
+        gas_price: u64, // TODO: Make this `u128`.
```

Comment:
Perhaps native question but is there a reason we can't just change the signature now? Does it cause a lot more cascading refactoring?

Reply:
Thanks for spotting, it was trivial to change this :)